src/entity/MultipleChoiceQuestion.java

```diff
@@ -0,0 +1,26 @@
+package entity;
+class MultipleChoiceQuestion extends Question {
+    private Map<int, String> answerOptions = new HashMap<>();
+    private int correctAnswer;
+
+    public MultipleChoiceQuestion(){}
+
+    public void setQuesion(String answer){
+        this.setQuesion(answer);
+    }
+    public String getQuesion(){
+        this.getQuesion();
+    }
+    public String getanswerOptions() {
+        return answerOptions;
+    }
+    public void setanswerOptions(Map<int, String> ansOps) {
+        this.answerOptions = ansOps;
+    }
+    public String getCorrectAnswer() {
```

Comment:
Why is this method called getCorrectAnswer in this class but getAnswer in the other class? Would it be better to have it both as getCorrectAnswer?